clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1006,6 +1036,9 @@ public String toEnrichedRst() {
 
         String lastKeyGroupName = "";
         for (ConfigKey key : sortedConfigs()) {
+            if (key.deprecated) {
+                continue;
```

Comment:
Still need to mark the ZK config as deprecated -- ie, need to update the call to `define` within `StreamsConfig` -- or will #2456 cover this part, too. \cc @cmccabe